src/module.ts

```diff
@@ -192,7 +192,7 @@ export default defineNuxtModule<ModuleOptions>({
         showURL: false
       }
     },
-    sources: ['content'],
+    sources: [],
```

Comment:
Shoudn't it becomes an object by default now?

```suggestion
    sources: {},
```